content/blog/2018/08/2018-08-13-simple-pull-request-plugin-final-evaluation.adoc

```diff
@@ -0,0 +1,149 @@
+
+. Test Multi-Branch Pipeline features support:
+.. Support for webhooks (link:https://issues.jenkins-ci.org/browse/JENKINS-51941[JENKINS-51941])
+.. Check if trusted people have approved a pull request and start build accordingly (link:https://issues.jenkins-ci.org/browse/JENKINS-52517[JENKINS-52517])
+. Finalize documentation (link:https://issues.jenkins-ci.org/browse/JENKINS-52518[JENKINS-52518])
+
+. Test the integration with various platforms Bitbucket, Gitlab, Github.
+
+
+=== Phase 3 evaluation presentation video
+
+
+Video: link:https://www.youtube.com/watch?v=GGEtN4nbtng[Link to video evaluation]
+++++
+<center>
+      <iframe width="720" height="400" src="https://www.youtube.com/watch?v=GGEtN4nbtng"
+  frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</center>
+++++
+
+=== Phase 3 evaluation presentation slides
+
+link:https://speakerdeck.com/player/46e467bc5e364cb08c39b9e4b69bdef2[Link to presentation slides]
+
+++++
+<center>
+    <iframe width="720" height="400" frameborder="0"
+      src="https://speakerdeck.com/player/46e467bc5e364cb08c39b9e4b69bdef2"></iframe>
+</center>
+++++
+
+=== My GSoC experience
+
+Student applications started on March 12 16:00 UTC and ended on March 27 16:00 UTC. Application period allowed me to explore
+many new technology and platforms that are making peoples life easy.
+
+Before starting of the application
+period I did not know anything about Jenkins. I found Jenkins organisation on the link:https://summerofcode.withgoogle.com/organizations/[GSoC organisations page]
+and came to know that I is a CI/CD platform that is used automate various things related to software development. I studied
+about Jenkins online and went through the link:https://jenkins.io/projects/gsoc/gsoc2018-project-ideas/[problem statements provided by some mentors].
+
+I decided that to work on link:https://jenkins.io/projects/gsoc/gsoc2018-project-ideas/#simple-pull-request-job-plugin[Simple Pull-Request Job Plugin] project.
+Then I wrote a draft proposal for this project and received many comments to refactor the proposal and enhance its quality from the mentors,
+then finally I submitted my link:https://summerofcode.withgoogle.com/serve/5141186688319488/[final proposal] to the Goolge.
+
+I was able to complete most of the tasks decided in Phase 1 and 2. After Phase 2 I was not able to give time to the project because
+of the placement season in the my college. I modified the code so that other plugin developers can contribute to it by Jenkins extensions.
+
+All the mentors made themselves available for most of the weekly calls and provided many valuable suggestions during the
+entire period of GSoC. There were sometimes when I was not able effectively. As communication is key while working remotely mentors
+suggested to communicate more thorough gitter chat.
+
+My overall experience of GSoC was good and all the mentors helped me as they can all times. This project allowed me to explore
```

Comment:
I think I follow what you're trying to say here, but these sentences could use a bit of cleanup:

> There were sometimes when I was not able effectively. As communication is key while working remotely mentors suggested to communicate more thorough gitter chat.
>
> My overall experience of GSoC was good and all the mentors helped me as they can all times.